src/main/java/reactor/kafka/receiver/internals/ConsumerEventLoop.java

```diff
@@ -360,6 +362,7 @@ boolean isPaused() {
         private final AtomicBoolean isPending = new AtomicBoolean();
         private final AtomicInteger inProgress = new AtomicInteger();
         private final AtomicInteger consecutiveCommitFailures = new AtomicInteger();
+        private boolean retrying;
```

Comment:
we check `this.retrying`, which is a simple `boolean`, in places where we also check `isPending` (`AtomicBoolean`). Should `retrying` also be an `AtomicBoolean` ?

Reply:
Good catch; thanks!